client/js/components/day/day.js

```diff
@@ -18,8 +18,9 @@ var Day = React.createClass({
 
   componentWillMount: function(){
     DayStore.addChangeListener(this._handleUpdate);
-     // trigger API call to get initial data
-    ViewActionCreator.getDayData();
+     // trigger API call to get initial data. Date.now will
+     // change once we get routing sorted
+    ViewActionCreator.getDayData(Date.now());
```

Comment:
Im starting to get a bit concerned we havent agreed on a standard date format through out application. We seem to be using different things in the SQL database, the frontend, and our server calls. I remember you spoke about a node library that would handle alot of these potential issues for us. Can we bring that in now and if not we should agree on a date format and make it clear in the docs